openapi.yml

```diff
@@ -586,6 +586,12 @@ components:
           type: string
           description: The exact user who should have ownership or be authorized to use the link.
           example: "Bill"
+        not_ownership:
+          type: array
+          items:
+            type: string
```

Comment:
@Ktmi, on `pathfinder` [we ended up with `array[T]`](https://github.com/kytos-ng/pathfinder/blob/master/openapi.yml#L204), since with the current use case we're expected only strings we might as well further constraint there too, right, can you send a PR? As it is on `pathfinder` it won't cause issues though.

Reply:
Done with kytos-ng/pathfinder#70